Source/WebGPU/WebGPU/Adapter.h

```diff
@@ -64,16 +64,17 @@ class Adapter : public WGPUAdapterImpl, public RefCounted<Adapter> {
     void makeInvalid() { m_device = nil; }
 
     Instance& instance() const { return m_instance; }
-
+    unsigned webProcessID() const { return m_webProcessID; }
```

Comment:
I don't think this belongs here :/

Reply:
oh maybe it should go in `Instance`? The name of that had me thinking there was only one `Instance` instance per GPU process. But it seems there is one `Instance` instance in the GPU process for each web process?